lib/page-object.rb

```diff
@@ -399,6 +401,10 @@ def self.register_widget(widget_tag, widget_class, base_element_tag)
 
   private
 
+  def root
+    @root_element || browser_root_for(browser, PageObject::Platforms.get)
```

Comment:
I wasn't sure if I should cache the html tag element as the root after gathering it, or if that would lead to stale element references, so I just let it search every time for now.